### PR TITLE
#5291 Create a special API to delete all objects with the same group

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5291)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
unrelated error

File: ohara-configurator/src/main/scala/oharastream/ohara/configurator/Configurator.scala [hunk 1]
```diff
@@ -236,7 +237,24 @@ class Configurator private[configurator] (val hostname: String, val port: Int)(
         ObjectRoute.apply,
         ContainerRoute.apply
       ).reduce[server.Route]((a, b) => a ~ b)
-    )
+    ) ~ pathPrefix(PrivateApi.PREFIX) {
+      delete {
+        import PrivateApi._
+        entity(as[Deletion]) { deletion =>
```
Comment (chia7712) [inline ohara-configurator/src/main/scala/oharastream/ohara/configurator/Configurator.scala:243]: @eechih Could you take a look?

*how to use this new API*

1. delete all connectors and topics which have group "g"

```DELETE /_private```
```json
{
  "groups": ["g"],
  "kinds": ["connector", "topic"]
}
``` 

2. delete all topics which have either group "g" or group "e" 

```DELETE /_private```
```json
{
  "groups": ["g", "e"],
  "kinds": ["topic"]
}
```

Comment (eechih) [reply ohara-configurator/src/main/scala/oharastream/ohara/configurator/Configurator.scala:243]: Seems good to me. Thanks @chia7712

Comment (chia7712) [reply ohara-configurator/src/main/scala/oharastream/ohara/configurator/Configurator.scala:243]: @eechih thanks for quick response. Let me merge it now.

**Pull request merged**